mlflow/models/evaluation/default_evaluator.py

```diff
@@ -79,7 +80,9 @@ def _infer_model_type_by_labels(labels):
 def _extract_raw_model(model):
     model_loader_module = model.metadata.flavors["python_function"]["loader_module"]
     if model_loader_module == "mlflow.sklearn" and not isinstance(model, _ServedPyFuncModel):
-        return model_loader_module, model._model_impl.sklearn_model
+        if isinstance(model._model_impl, _SklearnModelWrapper):
+            return model_loader_module, model._model_impl.sklearn_model
+        return model_loader_module, model._model_impl
```

Comment:
Can we just return 
```suggestion
        return model_loader_module, model._model_impl
```

Doesn't this work?

Reply:
If user wants to call some other functions on the raw model, like other than 'predict, predict_prob, or score', functions we didn't wrap in _SklearnModelWrapper, then this will be a problem

Reply:
`_model_impl` is a private field, we don't expect users to touch it.